Makefile

```diff
@@ -76,6 +76,13 @@ show-docker-images: ## Show all docker images and versions used in the helm char
 		2>/dev/null \
 		| gawk '/image: / {match($$2, /(([^"]*):[^"]*)/, a) ; printf "https://%s %s\n", a[2], a[1] ;}' | sort -u | column -t
 
+.PHONY: show-docker-images-no-link
+show-docker-images-no-link: ## Show all docker images and versions used in the helm chart without links
```

Comment:
This outputs a lot of duplicates. Also this could just be `make show-docker-images | awk '{print $2}'`

Reply:
Done